agent/src/agent/pipeline/config/stages/source/influx2.py

```diff
@@ -62,32 +62,22 @@ def _get_filter_condition(self) -> str:
         return filter_condition
 
 
-class TestInflux2Source(JythonSource):
+class TestInflux2Source(Influx2Source):
     JYTHON_SCRIPT = 'influx2.py'
     JYTHON_SCRIPTS_DIR = os.path.join(JythonSource.JYTHON_SCRIPTS_DIR, 'test_pipelines')
 
     def _get_script_params(self) -> list[dict]:
-        return [
-            {
-                'key': 'URL',
-                'value': self._get_url()
-            },
-            {
-                'key': 'HEADERS',
-                'value': self._get_headers()
-            },
+        params = super()._get_script_params()
```

Comment:
I don't think you should call the parent method here, query and monitoring_url from it are not used, just provide its own list of params and you also won't need to overwrite _get_query()

Reply:
Updated